config.json

```diff
@@ -14,6 +14,7 @@
   ],
   "ignored": [
     "bin"
```

Comment:
Actually, now that GitHub lets owners push to the PR branch, let me just fix this :)